### DISCUSSION TITLE
Cutting GoPro with data stream (GPS)

### DISCUSSION THREAD

**Discussion by Trigve** [mifi/lossless-cut#2003] [Q&A] [answered]
Hi,
I've come across the #1438. I'm using lossless-cut for cutting the GoPro videos with embedded GPS data. So according to the issue the data tracks aren't cut? I'm asking because I've encountered weird issue with telemetry overlay, where video goes out of sync from the telemetry data.
So my question is, when cutting the part of the video out with data stream, are the streams also cut out or they're just somehow preserved?

**Comment by mifi** [comment, marked as answer]
hi. that's how I understand it yes. I think data tracks cannot be cut because ffmpeg cannot decipher them. AFAIK `data` just means that it's some unprocessable data, and ffmpeg will just pass it along unaltered. you can try to disable the data streams (or any non-video/audio streams) to verify whether that's the cause of the de-sync

**Reply by Trigve** [reply]
Thank you for the reply.
I think what you said is true, because I've tried uncut video with telemetry overlay and it works OK. I cannot disable streams, because streams contains the GPS data which is used by telemetry overlay and the streams were causing havoc there.

So I think I need to change my workflow. First I would need to generate the overlay videos and then cut it.

**Reply by Trigve** [reply]
Just one question. Does this also applies for split video? That is if I split the video in the half, are the data streams split or they're somehow copied without being split?

**Reply by mifi** [reply]
I think the same applies yes because a split is basicaly just a cut happening 2 times.
